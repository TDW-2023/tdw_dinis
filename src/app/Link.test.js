import React from 'react';
import renderer from 'react-test-renderer';
import MeuComponente from './test_page';

test('snapshot para  o Componente de teste', () => {
  const component = renderer.create(<MeuComponente />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

