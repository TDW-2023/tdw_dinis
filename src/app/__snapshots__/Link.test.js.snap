// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot para  o Componente de teste 1`] = `
<div>
  <h1>
    Ol√°, mundo!
  </h1>
</div>
`;
